app/service/base_service.py

```diff
@@ -44,6 +44,10 @@ def jitter(fraction):
         i = fraction.split('/')
         return randint(int(i[0]), int(i[1]))
 
+    @staticmethod
+    def jitter_range(min, max):
+        return randint(int(min), int(max))
```

Comment:
can you use the jitter function above?

Reply:
I could if I combine the integers into a string... I want to store the sleep as two integers in the DB, because storing in fraction form makes things more challenging in other areas (max_sleep would be need to be calculated for Untrusted Hosts, and other areas).

Reply:
I'm using a format string and removed that function.